ingests/un_sdg.py

```diff
@@ -0,0 +1,153 @@
+import json
+import yaml
+import pandas as pd
+import tempfile
+import requests
+import datetime as dt
+import os
+
+from structlog import get_logger
```

Comment:
I've noticed `structlog` is not installed in walden, would you mind running `poetry add structlog` and committing changes here?